pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: test
-version: 0.12.24+5
+version: 0.12.24+6
```

Comment:
Do we need to update the minimum sdk version as well?

Reply:
No. This change should maintain the same semantics.